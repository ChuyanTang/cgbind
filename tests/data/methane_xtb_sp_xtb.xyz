5

C    0.00100    -0.00420   -0.01100 
H    -0.68840   -0.28620   -0.82170 
H    0.28110    1.06720    -0.06110 
H    -0.52550   -0.16990   0.94970  
H    0.93180    -0.60700   -0.05600 
