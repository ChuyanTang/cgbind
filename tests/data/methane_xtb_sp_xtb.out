      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. MÃ¤rz, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/07/04 at 18:17:45.977     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/tom/.local/bin/xtb methane_xtb_sp_xtb.xyz --chrg 0
          coordinate file            : methane_xtb_sp_xtb.xyz
          omp threads                :                     1
          number of atoms            :                     5
          number of electrons        :                     8
          charge                     :                     0
          spin                       :                   0.0
          first test random number   :      0.24507521923231

########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   8          :
          :  # atomic orbitals                   8          :
          :  # shells                            6          :
          :  # electrons                         8          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -4.2189123 -0.421891E+01  0.251E+00   16.45       0.0  T
   2     -4.2329767 -0.140645E-01  0.911E-01   16.27       1.0  T
   3     -4.2331072 -0.130437E-03  0.509E-01   16.17       1.0  T
   4     -4.2331602 -0.530752E-04  0.103E-01   16.05       1.0  T
   5     -4.2331603 -0.349976E-07  0.458E-03   16.05       9.8  T
   6     -4.2331603 -0.495094E-07  0.509E-04   16.05      87.8  T
   7     -4.2331603 -0.130568E-09  0.135E-04   16.05     331.5  T

   *** convergence criteria satisfied after 7 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.5776238             -15.7179
         2        2.0000           -0.4685447             -12.7498
         3        2.0000           -0.4657986             -12.6750
         4        2.0000           -0.4622945             -12.5797 (HOMO)
         5                          0.1274007               3.4668 (LUMO)
         6                          0.1753920               4.7727
         7                          0.1909618               5.1963
         8                          0.2085185               5.6741
      -------------------------------------------------------------
                  HL-Gap            0.5896953 Eh           16.0464 eV
             Fermi-level           -0.1674469 Eh           -4.5565 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.000 sec (  6.012%)
 Dispersion                     ...        0 min,  0.000 sec (  1.301%)
 integral evaluation            ...        0 min,  0.000 sec ( 12.780%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  1.197%)
 iterations                     ...        0 min,  0.000 sec ( 53.071%)
 molecular gradient             ...        0 min,  0.000 sec ( 19.105%)
 printout                       ...        0 min,  0.000 sec (  4.138%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.173366594192 Eh    ::
         :: gradient norm              0.033090159151 Eh/a0 ::
         :: HOMO-LUMO gap             16.046425136209 eV    ::
         ::.................................................::
         :: SCC energy                -4.233160314447 Eh    ::
         :: -> isotropic ES            0.001838533028 Eh    ::
         :: -> anisotropic ES          0.002815130158 Eh    ::
         :: -> anisotropic XC          0.004200171796 Eh    ::
         :: -> dispersion             -0.000665081761 Eh    ::
         :: repulsion energy           0.059793722617 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.5776238             -15.7179
         2        2.0000           -0.4685447             -12.7498
         3        2.0000           -0.4657986             -12.6750
         4        2.0000           -0.4622945             -12.5797 (HOMO)
         5                          0.1274007               3.4668 (LUMO)
         6                          0.1753920               4.7727
         7                          0.1909618               5.1963
         8                          0.2085185               5.6741
      -------------------------------------------------------------
                  HL-Gap            0.5896953 Eh           16.0464 eV
             Fermi-level           -0.1674469 Eh           -4.5565 eV

     #   Z        covCN         q      C6AA      Î±(0)
     1   6 C      3.689    -0.147    23.544     6.945
     2   1 H      0.923     0.039     2.465     2.455
     3   1 H      0.922     0.036     2.503     2.474
     4   1 H      0.922     0.035     2.517     2.481
     5   1 H      0.922     0.037     2.492     2.469

 Mol. C6AA /auÂ·bohrâ¶  :        124.380023
 Mol. C8AA /auÂ·bohrâ¸  :       2302.613267
 Mol. Î±(0) /au        :         16.823834


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  C   3.994        H    4 0.999    H    3 0.999    H    5 0.999    H    2 0.998
     2  H   0.998        C    1 0.998
     3  H   0.999        C    1 0.999
     4  H   0.999        C    1 0.999
     5  H   0.999        C    1 0.999

molecular dipole:
                 x           y           z       tot (Debye)
 q only:       -0.002      -0.001      -0.002
   full:       -0.003       0.002       0.005       0.016
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:        0.016       0.004      -0.006       0.010       0.013      -0.010
  q+dip:        0.122       0.036      -0.032       0.014       0.084      -0.090
   full:        0.075       0.022      -0.021       0.012       0.053      -0.055


           -------------------------------------------------
          | TOTAL ENERGY               -4.173366594192 Eh   |
          | GRADIENT NORM               0.033090159151 Eh/Î± |
          | HOMO-LUMO GAP              16.046425136209 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/07/04 at 18:17:45.984     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.007 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.007 sec
 * ratio c/w:     0.997 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.001 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.001 sec
 * ratio c/w:     0.999 speedup

